deepchecks/nlp/utils/text_properties.py

```diff
@@ -39,6 +39,7 @@
 MODELS_STORAGE = pathlib.Path(__file__).absolute().parent / '.nlp-models'
 FASTTEXT_LANG_MODEL = 'https://dl.fbaipublicfiles.com/fasttext/supervised-models/lid.176.bin'
 DEFAULT_SENTENCE_SAMPLE_SIZE = 300
+MAX_CHARS = 720  # Bert accepts max of 512 tokens, and experimentally 720 chars seems to work while 1024 doesn't
```

Comment:
I mean.
Technically, a user can have a string where each character is a token.

Reply:
yeah this is more a patch then a complete solution

Reply:
will do